js/textInputModel.js

```diff
@@ -166,10 +166,9 @@ class TextInputModel extends QuestionModel {
   checkAnswerIsCorrect(possibleAnswers, userAnswer) {
     const uAnswer = this.cleanupUserAnswer(userAnswer);
 
-    const answerIsCorrect = possibleAnswers.some(cAnswer => {
+    return possibleAnswers.some(cAnswer => {
       return this.cleanupUserAnswer(cAnswer) === uAnswer;
     });
-    return answerIsCorrect;
   }
 
   cleanupUserAnswer(userAnswer) {
```

Comment:
This should be `cleanupAnswer(answer)` if you're using it for configured and user answers.

Reply:
Removed this change in favour of a simplified map approach